src/test/multipleTestSuiteOptionSets.ts

```diff
@@ -0,0 +1,4 @@
+import { setTestSuiteOptions } from '../index'
```

Comment:
```suggestion
This file can be used to test `setTestSuiteOptions` manually, because these things are difficult to write automated tests for.

You can run this file to see the behavior if you set test suite options multiple times, and it should throw an error.

import { setTestSuiteOptions } from '../index'
```

Reply:
2b17ba1